lib/api.js

```diff
@@ -340,6 +343,11 @@ function _addNewLedgerNode(actor, options, callback) {
 function _addNewLedgerAgent(actor, ledgerNodeId, options, callback) {
   const laUuid = uuid();
   const routes = config['ledger-agent'].routes;
+
+  // FIXME: should these service URLs be hard coded into the record?  This
+  // does not allow for config changes to be applied easily.  Perhaps
+  // these services could be generated dynamically in the `get` API
```

Comment:
+1 to generated dynamically

Reply:
changed fixme to a todo and if we can get this commited, I'll add an issue as well.